masterlist.yaml

```diff
@@ -5724,3 +5746,58 @@ plugins:
         util: 'FO4Edit v4.0.3'
   - name: 'Better VATS - 0.75.esp'
     msg: [ *alreadyIn_WSE_BLD_Patch ]
+
+  - name: 'Farm.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/16810' ]
+    after:
+      - 'Better Cooking Stations.esp'
+
+  - name: 'Farm_CP_ScrapEverything_\w+\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/16810' ]
+    group: *scrapGroup
+
+  - name: 'Farm_WSPOverride_\w+\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/16810' ]
+    group: *dynamicPatchesGroup
```

Comment:
My only question is, how do you expect LOOT to sort these plugins?

I'm assuming `Farm_CP_ScrapEverything` plugins do have the relevant Scrap everything plugins as master files and that these patches need to be sorted after all other Scrap everything patches.

Should `Farm_WSPOverride` plugins be before or after bashed patches?

Reply:
If these plugins "Farm_CP_ScrapEverything" are not in "scrapGroup" group, you will see a message about cyclical rules.

"Farm_WSPOverride" plugins have no relation to bashed patches. Is "WorkshopParent "Workshop Parent Quest" [QUST:0002058E]" Override

Reply:
I know mods that overwrite WorkshopParent quest:
NorthlandDiggers Resources - Animals - Landscape - Garden and more...
Beast Master - Creature Companion Overhaul
Start Me Up - Alternate Start and Dialogue Overhaul

Start Me Up in "altStartGroup"

Reply:
> If these plugins "Farm_CP_ScrapEverything" are not in "scrapGroup" group, you will see a message about cyclical rules.

There are only two plugins in that group `Scrap Everything - Core.esp` and `Scrap Everything - Ultimate Edition.esp`.
`Scrap Everything - Core.esp` is included in `Scrap Everything - Ultimate Edition.esp`.
There's no set order between them since they shouldn't be used together.
You'll receive a cyclic sorting error if LOOT can't decide which plugin should win.
A sorting error will happen if you have patches for both `Scrap Everything - Core.esp` and `Scrap Everything - Ultimate Edition.esp`.

> "Farm_WSPOverride" plugins have no relation to bashed patches. Is "WorkshopParent "Workshop Parent Quest" [QUST:0002058E]" Override

Right so the idea is to make sure nothing else overrides it. Any idea of mods that this is incompatible with?

> Start Me Up - Alternate Start and Dialogue Overhaul - resolve by load order (Override LOOT)

I'm considering moving Alternate Start to an earlier group, Is placing this mod after it still a requirement?

Reply:
I checked again. Cyclical rules occur when Scrap Everything is installed incorrectly. When both plug-ins "Scrap Everything - Ultimate Edition.esp" and "Scrap Everything - Core.esp" are installed simultaneously. This is my setup for creating compatibility patches.
FOMOD installs both "Farm_CP_ScrapEverything_UE.esp" and "Farm_CP_ScrapEverything_Core.esp", then Vortex shows cyclical rules warning.
Correct installation 
"Scrap Everything - Ultimate Edition.esp" + "Farm_CP_ScrapEverything_UE.esp"
or
"Scrap Everything - Core.esp" + "Farm_CP_ScrapEverything_Core.esp"
does not cause an warning.
I removed "scrapGroup"

Reply:
Okay thank for confirming.
You can resolve that error if you need to, by adding an after to place `Scrap Everything - Core.esp` after `Scrap Everything - Ultimate Edition.esp` or vice versa.

Reply:
> You can resolve that error if you need to, by adding an after to place `Scrap Everything - Core.esp` after `Scrap Everything - Ultimate Edition.esp` or vice versa.

This is my setup for creating compatibility patches. I know this is the wrong setup. It's just faster to make compatibility patches. 
Next time, I will check LOOT sorting with the correct setup. :)